app/models/experimentation/stale_patient_experiment.rb

```diff
@@ -41,9 +41,12 @@ def eligible_patients(date)
       self.class.superclass.eligible_patients.where(id: sql.values)
     end
 
-    def memberships_to_notify(date)
-      # Patients who were enrolled on the `date`.
-      # To be implemented in a follow up PR.
+    # Patients who were enrollment date falls on
+    # one of the reminder template's remind_on days since `date`.
+    def memberships_to_notify(reminder_template, date)
+      treatment_group_memberships.status_enrolled.where(
+        experiment_inclusion_date: date - reminder_template.remind_on_in_days.days
+      )
```

Comment:
Why are we passing in the reminder template here? An experiment method getting a reminder_template as an argument feels odd. The experiment already knows about its reminder templates.

In addition, it seems like we're getting _all_ members who are eligible to be notified the day of the given reminder template, even if they're not sent _that_ reminder template according to their treatment group?

Can we invert this to return the result with `reminder_template`s included instead?

Reply:
Addressed in 0c6a229d8. This seems much better.